src/main/kotlin/org/malachite/estella/services/InterviewService.kt

```diff
@@ -42,18 +42,45 @@ class InterviewService(
     fun getLastInterviewIdFromApplicationId(applicationId: Int): PayloadUUID =
         getLastInterviewFromApplicationId(applicationId).getId()
 
-    fun getLastInterviewFromApplicationId(applicationId: Int): Interview =
+    fun getLastInterviewFromApplicationId(
+        applicationId: Int,
+        withPossibleHosts: Boolean = false
+    ): InterviewWithPossibleHosts =
         withExceptionThrower {
             getAllByApplicationId(applicationId).sortedBy { it.dateTime }.sortedWith { a, b ->
                 when {
                     a.dateTime == null -> -1
                     b.dateTime == null -> 1
                     else -> a.dateTime.compareTo(b.dateTime)
                 }
-
             }.first()
+        }.let { interview ->
+            if (withPossibleHosts) {
+                val possibleHosts = interview.applicationStage.application.applicationStages.first().stage.id.let {
+                    recruitmentProcessService.getProcessFromStage(it!!)
+                }.offer.creator.organization.id.let {
+                    hrPartnerService.getAllHRsFromOrganization(it!!)
+                }.map { it.user.mail }
+
+                interview.toInterviewWithPossibleHosts(possibleHosts)
```

Comment:
pierdoła, ale może ładniej by było żeby wołać to tylko raz poza ifem, a do possibleHostów przypisać pustą liste nullcheckiem?

Reply:
Jeśli przypisałbym pustą listę, na froncie nikt nie mógłby być hostem na spotkaniu. Tak to działa w MOC, że Select ma wartości z listy, jeśli lista nie jest nullem. W tym wypadku byłaby pustą listą, a pusta lista w JSie ewaluuje się to `true`, więc byłby pusty select.

Reply:
to w takim razie można tam  przypisać nulla, ale może o taki drobiazg nie warto xD

Reply:
Znaczy still wydaje mi się, że if lepszy, jako że nie trzeba będzie wykonywać dodatkowych operacji na bazie, kiedy o hostów pytania nie będzie XD